proto/ligato/vpp/interfaces/interface.proto

```diff
@@ -380,4 +382,23 @@ message IPIPLink {
     string src_addr = 2;
     // Destination VTEP IP address
     string dst_addr = 3;
+}
+
+message WireguardLink {
+  uint32 user_instance = 1;
+
+  // Interface index
+  uint32 interface = 2;
+
+  // Generate key automatically
+  bool generate_key = 3;
```

Comment:
Note that I might not have enough info to make this assumption, but could this field be possibly unnecessary and we could assume it is true if `private_key` is empty string? Does wireguard interface without private key make any sense? If it does not and must have private key defined or have generate key set to true then it might be simpler to just automatically set key to be generated if it is not set.

CC @milanlenco

Reply:
Seems like it really doesn't make sense.
Removed `generate_key` field